css/photoswipe.css

```diff
@@ -0,0 +1,406 @@
+/*! PhotoSwipe main CSS by Dmytro Semenov | photoswipe.com */
```

Comment:
We should be able to include this via npm from `node_modules` and have Parcel bundle it for us.